src/Hadolint/Rules.hs

```diff
@@ -311,33 +311,23 @@ useAdd = instructionRule code severity message check
     code = "DL3010"
     severity = InfoC
     message = "Use ADD for extracting archives into an image"
-    check (Copy src dst) = and [not (format `isSuffixOf` src) | format <- archive_formats]
-    check _ = True
-    archive_formats = [".tar", ".gz", ".bz2", "xz"]
-
-exposeMissingArgs = instructionRule code severity message check
```

Comment:
this is now caught as a parse error

Reply:
Can you please also remove it from `README.md`?